packages/mobile/src/localization/dayjs-ua/dayjs-ua.js

```diff
@@ -0,0 +1,98 @@
+import dayjs from 'dayjs';
```

Comment:
I don't think we need this, 'cause dayjs do have `uk` locale out of the box

Reply:
I was looking for 'ua' instead 'uk'